python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -7,207 +7,200 @@
 {%                         endif %}
 {%                     endif %}
+{#                     -- DSCP -- #}
+{%                     if match.actions.dscp is arista.avd.defined %}
+            set dscp {{ match.actions.dscp }}
+{%                     endif %}
+{#                     -- Traffic Class -- #}
+{%                     if match.actions.traffic_class is arista.avd.defined %}
+            set traffic class {{ match.actions.traffic_class }}
+{%                     endif %}
+{%                 endif %}
 {# END OF MATCH Section #}
-         !
       !
-{%                 endfor %}
-{%             endif %}
+{%             endfor %}
+{%         endif %}
 {# DEFAULT MATCH Section #}
-{%             if policy.default_actions is arista.avd.defined %}
-{%                 for version in policy.default_actions %}
-      match {{ version | lower }}-all-default {{ version | lower }}
+      match ipv4-all-default ipv4
+{%         if policy.default_actions.ipv4 is arista.avd.defined %}
          actions
-{%                     if version is arista.avd.defined('ipv4') %}
-{#                         -- COUNT packets ipv4 -- #}
-{%                         if policy.default_actions.ipv4.count is arista.avd.defined %}
+{#             -- COUNT packets ipv4 -- #}
+{%             if policy.default_actions.ipv4.count is arista.avd.defined %}
             count {{ policy.default_actions.ipv4.count }}
-{%                         endif %}
-{#                         -- Traffic Class ipv4 -- #}
-{%                         if policy.default_actions.ipv4.traffic_class is arista.avd.defined %}
-            set traffic class {{ policy.default_actions.ipv4.traffic_class }}
-{%                         endif %}
-{#                         -- DSCP ipv4 -- #}
-{%                         if policy.default_actions.ipv4.dscp is arista.avd.defined %}
-            set dscp {{ policy.default_actions.ipv4.dscp }}
-{%                         endif %}
-{#                         -- DROP Action ipv4 -- #}
-{%                         if policy.default_actions.ipv4.drop is arista.avd.defined(true) %}
+{%             endif %}
+{#             -- DROP Action ipv4 -- #}
+{%             if policy.default_actions.ipv4.drop is arista.avd.defined(true) %}
             drop
-{#                             -- LOGGING ipv4 only if DROP is set-- #}
-{%                             if policy.default_actions.ipv4.log is arista.avd.defined(true) %}
+{#                 -- LOGGING ipv4 only if DROP is set-- #}
+{%                 if policy.default_actions.ipv4.log is arista.avd.defined(true) %}
             log
-{%                             endif %}
-{%                         endif %}
-{%                     elif version is arista.avd.defined('ipv6') %}
-{#                         -- COUNT packets ipv6 -- #}
-{%                         if policy.default_actions.ipv6.count is arista.avd.defined %}
+{%                 endif %}
+{%             endif %}
+{#             -- DSCP ipv4 -- #}
```

Comment:
s1-leaf1(config-traffic-policy-match-abc-abc-ipv4)#protocol neighbors bgp enforce ttl maximum-hops

This support is missing in current version. Is it fine to add in this PR?